### ISSUE TITLE
GLES Version for Linux phones?

### ISSUE THREAD

**Issue by Justinzobel** [flathub/app.organicmaps.desktop#24]
I tried this flatpak on my desktop and it works great. I also tried it on my postmarketOS phone and I get errors saying it can't find GL:

```
GLES3_DEBUG_RECT_VSH > Shader compile error : 0:1(1): error: GLSL 1.50 is not supported. Supported versions are: 1.00 ES, 3.00 ES, 3.10 ES, and 3.20 ES
```

**Comment by fgaz** [comment]
What's the output of `glxinfo | grep version`?

**Comment by Justinzobel** [comment]
```
server glx version string: 1.4
client glx version string: 1.4
GLX version: 1.4
    Max core profile version: 4.5
    Max compat profile version: 4.5
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.2
OpenGL core profile version string: 4.5 (Core Profile) Mesa 22.3.3
OpenGL core profile shading language version string: 4.50
OpenGL version string: 4.5 (Compatibility Profile) Mesa 22.3.3
OpenGL shading language version string: 4.50
OpenGL ES profile version string: OpenGL ES 3.2 Mesa 22.3.3
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
    GL_EXT_shader_implicit_conversions, GL_EXT_shader_integer_mix, 
```

**Comment by fgaz** [comment]
Looks unrelated to how it's packaged through flatpak, so I reported the issue upstream: organicmaps/organicmaps#4373

**Comment by Ferenc-** [comment]
@Justinzobel as long as you use wayland, it should work on PmOS with the freshly released flatpak `2023.05.08`.
I have tested with Phosh, the GLSL version was correctly selected, if you still happen to have issues somehow, then please comment on [the upstream issue](https://github.com/organicmaps/organicmaps/issues/4373).
sxmo or other legacy X based setups are unlikely to get support.
